docs/data-management/post-session.md

```diff
@@ -374,11 +374,53 @@ ses-01
         └── sub-pilot_ses-01_task-rest_physio.tsv.gz
 ```
 
-
 ### Incorporate into version control with DataLad
 
 !!!info "Initiating the version-controled dataset"
 
-    Once at the beginning of the project, the datalad dataset will be created:
+    Once at the beginning of the project, the DataLad[1] dataset will be created:
 
     - [ ] Add stockage horus as an SSH remote.
+    - [ ] The preprocessing will be executed on an HPC. If datalad is not installed on the HPC, the most convenient user-based installation can be achieved by using Conda. 
```

Comment:
Not sure this is relevant: ideally, it shouldn't matter where you run things. Also, I would prefer to defer the configuration of the execution platform to other documentation sources (such as the DataLad handbook).